include/storage/S3FileLoader.h

```diff
@@ -0,0 +1,133 @@
+#pragma once
+
+#include <conf/FaasmConfig.h>
+#include <faabric/util/config.h>
+#include <faabric/util/func.h>
+#include <storage/FileLoader.h>
+#include <storage/S3Wrapper.h>
+
+namespace storage {
+class S3FileLoader final : public FileLoader
```

Comment:
The very verbose `FileLoader` interface is something I'll simplify in a later PR.